HeterogeneousCore/CUDACore/interface/CUDAScopedContext.h

```diff
@@ -23,8 +23,12 @@ namespace impl {
   public:
     int device() const { return currentDevice_; }
 
-    cuda::stream_t<>& stream() { return *stream_; }
-    const cuda::stream_t<>& stream() const { return *stream_; }
+    // cudaStream_t is a pointer to a thread-safe object, for which a
+    // mutable access is needed even if the CUDAScopedContext itself
+    // would be const. Therefore it is ok to return a non-const
+    // pointer from a const method here.
+    cudaStream_t stream() { return stream_->id(); }
```

Comment:
same as above, shouldn't we remove the non-`const` method ?

Reply:
I agree